.github/workflows/hosted-notebook-samples-check.yml

```diff
@@ -17,14 +17,18 @@ jobs:
         id: prepare
         run: |
           FILES=$(git diff origin/${{ github.event.pull_request.base.ref }} --name-only)
+          FILES=$(echo $FILES | awk -F\" '{OFS="\"";for(i=2;i<NF;i+=2)gsub(/ /,"%20",$i);print}')
```

Comment:
This change is so that files with spaces in the names are also checked to be tested